tests/integration/targets/dcnm_vrf/tasks/main.yaml

```diff
@@ -9,35 +10,38 @@
     method: GET
     path: /appcenter/cisco/ndfc/api/about/version
   register: result
-  ignore_errors: yes
+  failed_when: false
```

Comment:
This does not look right to me.  Not sure why `ignore_errors` was changed to `failed_when`